plugins/woocommerce-blocks/tests/e2e/tests/single-product-details/single-product-details.block_theme.spec.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { expect, test } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+
+const blockData = {
+	name: 'Product Details',
+	slug: 'woocommerce/product-details',
+};
+
+test.describe( `${ blockData.slug } Block`, () => {
+	test( "block can't be inserted in Post Editor", async ( {
+		editorUtils,
+		admin,
+	} ) => {
+		await admin.createNewPost();
+		await expect(
+			editorUtils.insertBlockUsingGlobalInserter( blockData.name )
+		).rejects.toThrow();
+	} );
+
+	test( 'block can be inserted in Site Editor', async ( {
+		editorUtils,
+		admin,
+	} ) => {
+		await admin.visitSiteEditor( {
+			postId: 'woocommerce/woocommerce//archive-product',
```

Comment:
This block is only available in the Single Product template.

Reply:
Fixed with [490b49c](https://github.com/woocommerce/woocommerce/pull/46671/commits/490b49c02ee991b696dd18331921f61de3f05508)